eclair-front/src/main/scala/fr/acinq/eclair/Boot.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2019 ACINQ SAS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.acinq.eclair
+
+import java.io.File
+
+import akka.actor.ActorSystem
+import com.typesafe.config.{ConfigFactory, ConfigParseOptions, ConfigSyntax}
+import grizzled.slf4j.Logging
+import kamon.Kamon
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+object Boot extends App with Logging {
+  try {
+    val datadir = new File(System.getProperty("eclair.datadir", System.getProperty("user.home") + "/.eclair"))
+    val config = ConfigFactory.parseString(
+      Option(System.getenv("AKKA_CONF")).getOrElse("").replace(";", "\n"),
+      ConfigParseOptions.defaults().setSyntax(ConfigSyntax.PROPERTIES))
+      .withFallback(ConfigFactory.load())
+
+    implicit val system: ActorSystem = ActorSystem("eclair-node", config)
```

Comment:
I still think this is worth a comment explaining that it really must be `eclair-node` and not `eclair-front`

Reply:
Fixed with c0e330a11.